### PR TITLE
UG: add CLI description

### PR DESCRIPTION
**Pull request by alientian**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -145,6 +145,73 @@ In this section, you will learn how to set up LinkMeIn in your computer.
 
 # Using LinkMeIn's CLI
 
+LinkMeIn uses the Command Line Interface (CLI). 
+CLI is a means of interacting with a computer program by **inputting lines of text called command-lines**.
+
+Do not be put off by its name! Read on to find out how simple CLI is in LinkMeIn.
+
+_Read a quick [introduction](#introducing-linkmeins-gui) to familiarise yourself with our user interface._
```
Comment (tayruxin) [inline docs/UserGuide.md:153]: Can don't need this? since they would have already read thru the previous section.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -145,6 +145,73 @@ In this section, you will learn how to set up LinkMeIn in your computer.
 
 # Using LinkMeIn's CLI
 
+LinkMeIn uses the Command Line Interface (CLI). 
+CLI is a means of interacting with a computer program by **inputting lines of text called command-lines**.
+
+Do not be put off by its name! Read on to find out how simple CLI is in LinkMeIn.
```
Comment (GlendaChong) [inline docs/UserGuide.md:151]: simple CLI is **for** LinkMeIn

File: docs/UserGuide.md [hunk 3]
```diff
@@ -145,6 +145,73 @@ In this section, you will learn how to set up LinkMeIn in your computer.
...
+
+_Read a quick [introduction](#introducing-linkmeins-gui) to familiarise yourself with our user interface._
+<div style="margin-top: 10px; margin-bottom: 10px">
+    <img src="images/cli-description/CliDescription1.png" alt="CliDescription1" width="650"/>
+</div>
+
+
+The Command box is where you input the command lines. After you press enter, LinkMeIn reads your input and replies you through the Message Box.
```
Comment (GlendaChong) [inline docs/UserGuide.md:159]: Command **Box**

File: docs/UserGuide.md [hunk 4]
```diff
@@ -145,6 +145,73 @@ In this section, you will learn how to set up LinkMeIn in your computer.
...
+</div>
+
+Let's use the [`add` command](#adding-a-company--add) format as shown below to explain the components. 
+<div style="margin-top: 10px; margin-bottom: 10px">
+    <img src="images/cli-description/CommandFormat2.png" alt="CommandFormat2" width="650"/>
+</div>
+
+`add` is the command word for add feature. Every command line must have a command word to indicate the feature you wish to use.
+The prefix specifies the parameter type. For example, `c/John` adds John as company name while `n/John` adds John as recruiter's name.
+Replace the `UPPER_CASE` parameters with your application details and you have successfully written a command line!
```
Comment (tayruxin) [inline docs/UserGuide.md:186]: Replace the parameters with your application details and you have successfully written a command!

File: docs/UserGuide.md [hunk 5]
```diff
@@ -145,6 +145,73 @@ In this section, you will learn how to set up LinkMeIn in your computer.
...
+- `add c/GovTech r/Software Engineer s/PA n/John Doe d/12-12-2023 e/johnDoe@gmail.com p/98765432 pr/medium`
+- `add c/GovTech r/Software Engineer s/PA n/John Doe d/12-12-2023 e/johnDoe@gmail.com p/98765432`
+
+<div markdown="block" class="alert alert-info">
+**:information_source: Notes:**<br>
+* Parameters can be in any order. e.g. if the command specifies c/COMPANY_NAME r/ROLE, r/ROLE c/COMPANY_NAME is also acceptable.
+* Parameters are case-insensitive. e.g. c/COMPANY_NAME is the same as c/company_name.
+</div>
+
+Notice that the `add` command does use `INDEX` in its command format. Each feature has a unique command format.
```
Comment (GlendaChong) [inline docs/UserGuide.md:207]: `add` command does **NOT**

File: docs/UserGuide.md [hunk 6]
```diff
@@ -523,12 +582,15 @@ To edit the company's information, type `edit 2 s/PI r/Frontend Developer`
...
-    **More examples:**
-    - `edit 3 d/11-09-2025 pr/LOW`
-        Edits dateline and priority of 3rd company in the displayed list.
-    - `edit 1 n/Amy Tan e/amytan@gmail.com`
-        Edits recruiter's name and email of 1st company in the displayed list.
 
+**More examples:**
+- `edit 3 d/11-09-2025 pr/LOW`
+
+  Edits dateline and priority of 3rd company in the displayed list.
```
Comment (GlendaChong) [inline docs/UserGuide.md:589]: ...priority of **the** 3rd....

File: docs/UserGuide.md [hunk 7]
```diff
@@ -523,12 +582,15 @@ To edit the company's information, type `edit 2 s/PI r/Frontend Developer`
...
+  
+- `edit 1 n/Amy Tan e/amytan@gmail.com`
+
+  Edits recruiter's name and email of 1st company in the displayed list.
```
Comment (GlendaChong) [inline docs/UserGuide.md:593]: ...email of **the** first ...

**Review by GlendaChong — APPROVED** [review]

**Pull request merged**